test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -144,7 +144,15 @@
                                        {:path "/"
                                         :type "path"
                                         :query-params {"env" [:notastring]}}
-                                       :allow "*"})))))
+                                       :allow "*"})))
+
+    (is (thrown? IllegalArgumentException
```

Comment:
I think this should be a `thrown-with-msg?` rather than a `thrown?` so that the message content is evaluated.  Also, I think the expected body of the message should be `#".* It should be either a string or list of strings .*"` - note the absent "a" between "or" and "list".  Could also change the error message thrown to include the "a".  Either way works for me.